### ISSUE TITLE
LC0003 For CodeunitId in AddAction of ErrorInfo

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#405]
Currently the rule LC0003  doesn't warn on AddAction  of ErrorInfo.
```al
local procedure ThrowError()
var
    MyErrorInfo: ErrorInfo;
begin
    MyErrorInfo:= ErrorInfo.Create('Test Error');
    MyErrorInfo.AddAction('Test Action', 50000, 'ShowRecord'); //Warning expected here
    end;
    Error(MyErrorInfo);
end;
```

**Comment by pri-kise** [comment]
I haven't check if Notificatin.AddAction is already checked.

**Comment by Arthurvdv** [comment]
Hi, over the past months I’ve been putting a lot of time and care into the successor of BusinessCentral.LinterCop: ALCops (https://alcops.dev/). Because of that, this repository is no longer actively maintained.

I’m closing this issue, as it won’t be handled here anymore. If this is still relevant for you, please have a look at ALCops/Analyzers (https://github.com/ALCops/Analyzers).

To make the transition easier, you can also follow the LinterCop Migration Guide: https://alcops.dev/docs/lintercop-migration/

If you have any questions or run into issues during the transition, feel free to reach out through ALCops. Happy to help 🙂
